app/controllers/api/v2/users_controller.rb

```diff
@@ -59,6 +59,7 @@ def show_current
         param :timezone, ActiveSupport::TimeZone.all.map(&:name), :required => false, :desc => N_("User's timezone")
         param :locale, FastGettext.available_locales, :required => false, :desc => N_("User's preferred locale")
         param :role_ids, Array, :require => false
+        param :mail_enabled, :bool, :desc => N_("Enable user E-mail")
```

Comment:
Sorry, the last nitpick: what about 
```suggestion
        param :mail_enabled, :bool, :desc => N_("Enable user's email")
```